sanitychecks.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2016 Tuenti Technologies S.L. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+        "log"
+        "strings"
+        "strconv"
+
+	"k8s.io/client-go/pkg/api/v1"
+        "github.com/achanda/go-sysctl"
+)
+
+var (
+        ephemeralPortsRange *EphemeralPortsRange
+)
+
+type EphemeralPortsRange struct {
+        check    bool
+        LowPort  int32
+        HighPort int32
+}
```

Comment:
You could define a method like  for this struct with the logic to check if a Range contains a port, so you could write checks like:
```
port := 2048
r := EphemeralPortsRange{check: true, LowPort: 1024, HighPort; 32000}
if r.Contains(port) {
    panic()
}